client/record/cli/sendtx.go

```diff
@@ -0,0 +1,82 @@
+package cli
+
+import (
+	"os"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/wire"
+	authcmd "github.com/cosmos/cosmos-sdk/x/auth/client/cli"
+	"github.com/irisnet/irishub/client/context"
+	"github.com/irisnet/irishub/client/utils"
+	"github.com/irisnet/irishub/modules/record"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+)
+
+const (
+	FlagFileName   = "file-name"
+	FlagTargetPath = "target-path"
+	FlagTxHash     = "tx-hash"
+)
+
+func GetCmdSubmit(cdc *wire.Codec) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "submit",
+		Short: "submit specified file",
```

Comment:
The ' record submit ' tx should handle the following two things:
1. upload the input file to the ipfs network
2. build a Record Msg tx with the ipfs hash and broadcast to the blockchain network.

It's not enough that just broadcast the ipfs hash.